Lamina/include/Math/matrix.hpp

```diff
@@ -0,0 +1,220 @@
+﻿#ifndef LM_MATRIX
+#define LM_MATRIX
+
+#include <iostream>
+#include <array>
+#include <vector>
+
+///////////////////////////////////////////////////////////////////////////
+// Row major matrix library with addition, subtraction, and multiplication.
+// 
+// To define a new matrix type:
+// typedef Matrix<type, columns, rows> name;
+// 
+// or directly use the matrix class:
+// Matrix<type, columns, rows> name(*matrix data*)
+///////////////////////////////////////////////////////////////////////////
+namespace lm
+{
+	//main matrix class
+	template<typename mat_type, size_t columns, size_t rows>
+	class Matrix
+	{
+	public:
+		//constructors
```

Comment:
I'd remove this comment since it's redundant. Also remove unnecessary semicolons below.